web/cobrands/tfl/assets.js

```diff
@@ -109,75 +50,10 @@ var red_routes_layer = fixmystreet.assets.add(defaults, {
     },
     name: "Red Routes",
     max_resolution: 9.554628534317017,
-    road: true,
     non_interactive: true,
     always_visible: true,
     all_categories: true,
-    nearest_radius: 0.1,
     stylemap: tlrn_stylemap,
-    no_asset_msg_id: '#js-not-tfl-road',
-    actions: {
-        found: function(layer, feature) {
-            fixmystreet.message_controller.road_found(layer, feature);
```

Comment:
As discussed, I think the first bit of found/not_found needs to run on the tfl cobrand alone, to keep its current behaviour.